src/equations/numerical_fluxes.jl

```diff
@@ -0,0 +1,31 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# This file contains general numerical fluxes that are not specific to certain equations
+
+# An empty version of the `min_max_speed_chen_noelle` function is declared here
+# in order to create a dimension agnostic version of `flux_hll_chen_noelle`.
+# The full description of this wave speed estimate can be found in the docstrings
+# for `min_max_speed_chen_noelle` in `shallow_water_1d.jl` or `shallow_water_2d.jl`.
```

Comment:
```suggestion
# for `min_max_speed_chen_noelle` in `shallow_water_wet_dry_1d.jl`
or `shallow_water_wet_dry_2d.jl`.
```

Reply:
Done in 77f44c650f95e4bc7bcbd284ecd2f60f6b9d0609